crates/uv/src/commands/tool/run.rs

```diff
@@ -139,6 +139,50 @@ pub(crate) async fn run(
         command.to_string_lossy(),
         args.iter().map(|arg| arg.to_string_lossy()).join(" ")
     );
+    if has_from {
```

Comment:
I think this should probably run regardless of whether the user explicitly passed `--from`. E.g., `uv tool run fastapi` should also show this warning, right?